charts/intel-gpu-resource-driver/templates/node-feature-rules.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.nfd.enabled }}
+{{- if or .Values.nodeFeatureRules.enabled .Values.nfd.enabled }}
```

Comment:
So node feature rules can be deployed when you already have an existing NFD running in the cluster.